qadence/backends/pulser/backend.py

```diff
@@ -71,12 +70,16 @@ def make_sequence(circ: QuantumCircuit, config: Configuration) -> Sequence:
         raise ValueError("Specified device is not supported.")
 
     max_amp = device.channels["rydberg_global"].max_amp
-    min_duration = device.channels["rydberg_global"].min_duration
 
     if config.spacing is not None:
         spacing = config.spacing
     elif max_amp is not None:
+        spacing = DEFAULT_SPACING
```

Comment:
Not sure why it is there if you reassign it later.

Reply:
Hmm I changed this, this line should not be there